tracer/time_windows.go

```diff
@@ -0,0 +1,31 @@
+package tracer
+
+import (
+	"golang.org/x/sys/windows"
+	"time"
+)
+
+// This method is more precise than the go1.8 time.Now on Windows
+// See https://msdn.microsoft.com/en-us/library/windows/desktop/hh706895(v=vs.85).aspx
+// It is however ~10x slower and requires Windows 8+.
```

Comment:
Agree a test would ne nice. Possibly tricky as all those timing tests are hard to achieve... Maybe only check that if we sleep a microsecond, the time elapsed is non-zero?